libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3729,7 +3766,9 @@ void GCS_MAVLINK::handle_common_mission_message(const mavlink_message_t &msg)
     // GCS request the full list of commands, we return just the number and leave the GCS to then request each command individually
     case MAVLINK_MSG_ID_MISSION_REQUEST_LIST:       // MAV ID: 43
     {
-        handle_mission_request_list(msg);
+        if (!is_high_latency_link) {
+            handle_mission_request_list(msg);
+        }
```

Comment:
So you can never get any mission items?

Hmmm.

Maybe this should be based on NOSTREAMOVERRIDE too?  If you squint it's kinda-sorta like a temporary stream....

Reply:
> So you can never get any mission items?

Correct. Mission packets would clag up the link. It's expected that the user has already loaded the mission prior (over a low latency link)

Reply:
Mission packets can now get through